php/class-fieldmanager-markdown.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Class file for Fieldmanager_Markdown
+ *
+ * @package Fieldmanager
+ */
+
+use Fieldmanager\Libraries\Parsedown;
+
+/**
+ * Static markdown field.
+ */
+class Fieldmanager_Markdown extends Fieldmanager_Content {
+
+	/**
+	 * Override field_class.
+	 *
+	 * @var string
+	 */
+	public $field_class = 'markdown';
+
+	/**
+	 * Render content as html from parsed markdown.
+	 *
+	 * @param mixed $value Unused value.
+	 * @return string Rendered content.
+	 */
+	public function render_content( $content ) {
+		return ( new Parsedown\Parsedown() )->text( $content );
```

Comment:
Also, should the value go through `wp_kses_post()`?

Reply:
I'm still a bit on the fence, but I think it's for the best.